### ISSUE TITLE
Dialogue for "Coalition: Expeditions Past 7" or "7B" should change if you have already accessed the Terminus wormhole

### ISSUE THREAD

**Issue by opusforlife2** [endless-sky/endless-sky#8031]
### Problem Description

The mission dialogue for "7" contains a reference to a "crimson anomaly", and there is a lot of such text in "7B". The player, however, has already travelled through this wormhole, and knows exactly what it is. But the dialogue contains no references to their knowledge.

Considering that other dialogues branch if you have knowledge of something (Coalition if you know Hai, Hai if you know Remnant, Wanderer if you know Hai), this stands out as an anomaly. (A real one, unlike the Terminus wormhole which the player already knows about.)

### Related Issue Links

N/A

### Desired Solution

There should be a branch in the dialogue, maybe as early as the first reference, if the player has travelled through the Terminus wormhole (or has visited Cardea, but that can be achieved via a Jump Drive).

At the very least, even if it doesn't change anything regarding the choices, it should talk about the player's knowledge of the "curious red cloud" being a wormhole that needs a Quantum Keystone to access.

A more involved dialogue might let the player tell Sedlitaris about what it actually is, with or without mentioning the Keystone. But that opens up a whole can of worms about keeping the Remnants secret. This problem doesn't arise in the Pond Strider mission because the Remnant can choose where to meet the Hai.

### Alternative Approaches

If a condition like that cannot be created in the mission, then a compromise would be if the player knows the Remnants.

If knowing the Remnants is taken as the basis for the dialogue, then the bare minimum that needs to be done is that the dialogue itself mentions that the player kept mum about the wormhole, to keep the Remnants secret as per their wishes.

But if the player is given the choice to explain about the Remnants, that might again create a problem with keeping the Remnants secret. The Ivan mission is based on this exact premise. Moreover, even if the player knows the Remnants, it doesn't come up as a dialogue choice with Quarg and Hai when the Coalition asks about other alien species.

**Comment by Arachi-Lover** [comment]
As I stated before on the Discord when this was brought up there, given this concerns the region Zitchas is responsible for, I'd first want his approval on any such action of revealing to Sedlitaris what the anomaly mentioned is.

**Comment by Zitchas** [comment]
I'll just point out that technically the Remnant are not, so far as the pilot knows, an alien species. Thus in such discussions the Remnant being mentioned are about on par with mentioning the Deep, or anarchist settlements that don't consider themselves part of the Republic, or other such groups.

There are a *lot* of places in ES that *could* have branches, and not to mention many places where the player theoretically could ask more questions or offer more information, but doesn't. I don't think it is particularly beneficial to add this in here, especially if there's no point to it. Leaving it as is prevents it from becoming a plot hole where the player asks "Well, why doesn't the coalition do something with this?" If the coalition is going to do something about it, then that's a conversation we need to have.

On a larger scale, the Ember Waste present a challenge: Almost every faction has at least some scientists and curious people; and theoretically they could *all* be interested in exploring the Ember Waste. However, the more factions that are expploring the Ember Waste, the more likely they are to run into each other, unless they all happen to hire the player to do their exploring for them and the player just neglects to mention that...

**Comment by Arachi-Lover** [comment]
I'll also add that the Saryds who first explored, and later Heliarch expeditions, would've naturally been curious as to the phenomenon, but as stated in the missions they can't get it to do anything/do anything to it, as they don't posess keystones.
The focus given to Terminus on the missions, on a meta level, is more to serve as a late-game pointer to the EW, for players that might not have gotten there, but have a JD and got to the Coalition.

**Comment by ThrawnCA** [comment]
I agree that the dialogue should at least acknowledge the player's knowledge of the wormhole, even if revealing it to the Coalition isn't on the table.

Of course, since the player then has the ability to fly Sedlitaris _through_ said wormhole and visit the Remnant, well...I guess she spends a lot of time in her bunk?

**Comment by ThrawnCA** [comment]
This is particularly poignant toward the end, in "Coalition: Expeditions Past 10", where Sedlitaris is lamenting not achieving anything; "Inconclusive reports on red nebulas or disappearing ruins, what good are they?" At that point, it would make a lot of sense for the player to have the option to speak up and tell her, "Actually, I do know more about that red nebula..."
